File: app/src/main/java/com/kacper/and_krakgo/helpers/ToastMessageHelper.java
package com.kacper.and_krakgo.helpers;

import android.content.Context;
import android.widget.Toast;

import com.kacper.and_krakgo.KrakGoApp;

/**
 * Created by kacper on 22/10/2017.
 */

public class ToastMessageHelper {
    public static void showToast(String textToShow, boolean isLong) {
        if (isLong) {
            Toast.makeText(KrakGoApp.getApplicationCtx(), textToShow, Toast.LENGTH_LONG).show();
        } else {
            Toast.makeText(KrakGoApp.getApplicationCtx(), textToShow, Toast.LENGTH_SHORT).show();
        }
    }

    public static void showShortToast(String textToShow) {
        showToast(textToShow, false);
    }

    public static void showShortToast(int resourceId) {
        if (KrakGoApp.getApplicationCtx() != null)
            showShortToast(KrakGoApp.getApplicationCtx().getString(resourceId));
    }
}

File: app/src/main/java/com/kacper/krakgo/helpers/ToastMessageHelper.kt
package com.kacper.krakgo.helpers

import android.content.Context
import android.widget.Toast

import com.kacper.krakgo.KrakGoApp

/**
 * Created by kacper on 22/10/2017.
 */

object ToastMessageHelper {
    private fun showToast(context: Context?, textToShow: String, isLong: Boolean) {
        if (isLong) {
            Toast.makeText(context, textToShow, Toast.LENGTH_LONG).show()
        } else {
            Toast.makeText(context, textToShow, Toast.LENGTH_SHORT).show()
        }
    }

    fun showShortToast(context: Context?, textToShow: String) {
        showToast(context, textToShow, false)
    }

    fun showShortToast(context: Context?, resourceId: Int) {
        if(context!= null) showShortToast(context, context.getString(resourceId))
    }
}
